### ISSUE TITLE
Does this patch data collection too?

### ISSUE THREAD

**Issue by DerKamii** [MysteryDash/Offline-PS4-Remote-Play#30]
Hey Buddy,

it's not an "issue" per se but rather a question:
Does this patch the "data collection" too?

I don't know what they're thinking in there data policy or why remote play needs to collect additional data...

Kind regards,
Kami

**Comment by MysteryDash** [comment]
Nope, the patch only prevents the Remote Play from auto-updating or requesting updates.

**Comment by MysteryDash** [comment]
It was first meant to be used offline, so I didn't really think about the data collection either anyway.

**Comment by J0NY13** [comment]
Hi :)
I wanted to ask if you know what is this number problem i get on remoteplay, i cant log in and previusly didnt had this problem o.O i tried looking for someone else happen this and dont for what i find o. o
![coco](https://user-images.githubusercontent.com/64482505/80499183-094bd400-8943-11ea-9bb9-797482364d3e.png)

**Comment by DerKamii** [comment]
For what reason do you want to login on with a offline patched remote connect?
This does sound counter intuitive.

@MysteryDash if you could spare the time, it would be grate if you look into it and disable data collection feature too. I don't care if i'm offline or not, i don't see why it should be neccasssary in the first place...

**Comment by J0NY13** [comment]
It ask me to log in when i want to conect to my PS4, this never happenede before thats why i try to log in again ._ . cause i dont know how to conect it without log in, there is no other option ._ .
If i click  manual mode that window show too ._ .
sorry to bother

**Comment by MysteryDash** [comment]
@J0NY13 Please create your own issue when your problem is not the same. Furthermore, your issue does not seem to be linked with the Remote Play itself, so try to login in your browser directly and see if it works.

@DerKamii Even if the Remote Play is patched for offline use, a first login is required to be able to connect to the PS4. The Offline Remote Play is useful if you are on the go or if your Internet connection is down for a long time, not if you never had a connection in the first place.

I will try to take a look at the telemetry.
